### ISSUE TITLE
Accessibility is not supported.

### ISSUE THREAD

**Issue by leolorenzoluis** [swimlane/ngx-datatable#863]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
Using keyboard navigations are not supported. For example, it cannot tab to the column headers and press enter/space to sort.

**Expected behavior**
Be able to navigate and use the table with keyboard if there's no mouse. This is required especially if we are complying with https://www.section508.gov/ 

**What is the motivation / use case for changing the behavior?**
https://www.section508.gov/

**Comment by Pranayku** [comment]
Having section508 support will really help. This is actually a deciding factor for us.

**Comment by amcdnl** [comment]
I can do some of it but its really difficult to handle this for virtual scrolling.

**Comment by brunano21** [comment]
I'm really interested in accessibility support and I might help you with that.

**Comment by brunano21** [comment]
Hi guys,
any news about this?

**Comment by ParadoxZero** [comment]
Hi! 
Is there any work going on about this?

**Comment by mitschmidt** [comment]
An important side note: I am going through a barrier check (assessment) in Germany right now with a recent project, and it turns out the `ngx-datatable` does not conform to screenreaders' requirements. As it does not make use of the `<th>` and `<tr>` html elements, the screenreader is not able to grab the information accordingly. I might need to switch to another solution for the tables in question, yielding a lot of extra work.

As this is related to accessibility I ended up posting here, but I might as well create a seperate issue for that?

Anybody else facing the same problem?

**Comment by mitschmidt** [comment]
Cool, could you post a link? Any chance that this is going to be released within the next weeks?

**Comment by AlexanderSchiff** [comment]
Updates?
